Source/Krypton Components/Krypton.Docking/Krypton.Docking 2022.csproj

```diff
@@ -39,6 +39,7 @@
         <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
         <Configurations>Debug;Release;Installer;Nightly;Canary</Configurations>
         <LangVersion>preview</LangVersion>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why ?

Reply:
Removed, was for Acrylic code, now commented out